src/client/components/pages/entities/author.js

```diff
@@ -64,6 +67,25 @@ function AuthorAttributes({author}) {
 					<dl>
 						<dt>Sort Name</dt>
 						<dd>{sortNameOfDefaultAlias}</dd>
+						<dt>Ratings</dt>
+						<dd>
+							<Rating
+								allowHalfIcon
+								readonly
+								allowHover={false}
+								className="rating-stars"
+								fillColor="#46433A"
+								initialValue={averageRating}
+								ratingValue={0}
+								size={20}
+								stars={5}
+							/>
+						</dd>
+						{reviewsCount ?
+							<dd className="small text-muted">
```

Comment:
You can also add the utility css class `mt-n2` (= "margin top negative 2") as well to have the text a bit closer to the stars:
```suggestion
							<dd className="mt-n2 small text-muted">
```

Before:
<img width="159" alt="image" src="https://user-images.githubusercontent.com/6179856/186897544-9bfdabd3-ce1f-4380-ac2d-79855e7dbcce.png">

After: 
<img width="196" alt="image" src="https://user-images.githubusercontent.com/6179856/186897497-5e56db8f-f7d4-4b73-80cf-8baabbad8327.png">

Reply:
This looks much neat. Done 👍